app/controllers/responses_controller.rb

```diff
@@ -8,4 +10,19 @@ def show
     response = Response.find params[:id]
     render json: ResponseSerializer.new(response).as_json
   end
+
+  def update
+    response_finished = FinishResponse.run response_uuid: params[:id]
+    if response_finished.valid?
+      head :ok
+    else
```

Comment:
Je hebt gelijk dat een exceptie hier wat lelijk is. Maar is het idee van ActiveInteraction niet dat het alleen weet heeft van je business logic en geen HTTP zaken zoals status codes?